docs/Components-EmptyState.md

```diff
@@ -0,0 +1,296 @@
+# EmptyState Component
+
+[Back to all components](./Components.md)
+
+The `EmptyState` component is to be used when there is no data to display, usually after attempting to load some content from an external API. It is a container that can hold a heading and content. It can also display an image and provide a button to interact with.
+
+## Props
+
+```tsx
+<EmptyState
+  preset="default"
```

Comment:
There's no `default` preset. I added one called `generic` mainly because it's actually not "default" (there's no defaults for EmptyState`.